packages/ember-htmlbars/tests/helpers/custom_helper_test.js

```diff
@@ -23,36 +23,6 @@ QUnit.module('ember-htmlbars: custom app helpers', {
   }
 });
 
-QUnit.test('dashed shorthand helper is resolved from container', function() {
```

Comment:
yes, we used to have a distinction between dashed and non-dashed, but that was removed in 1.13.